### DISCUSSION TITLE
Unable to get info on how to build this project on Ubuntu.

### DISCUSSION THREAD

**Discussion by pankaj-54666** [nlohmann/json#2389] [Q&A] [answered]
Hello,

I read through the documentation, but i am unsure as how to build this project.
I want to build this project on Ubuntu machine. 

In README section, i can see only ***Execute unit tests**, which i guess need to run after building of project with changes.

Could you please point out as, how to build this project.

**Comment by nlohmann** [comment, marked as answer]
The library is header only - there is nothing to build. See https://github.com/nlohmann/json#integration on how to integrate it into your project.

**Reply by pankaj-54666** [reply]
I was searching for how to build the project after changes in the source code.

I think the relevant steps is as per: 
Step_1) Edit Files inside folder `include/nlohmann/` 
Step_2) Generate `single_include/nlohmann/json.hpp`
Step_2) Run unit test

Reference Link:https://github.com/nlohmann/json/blob/develop/.github/CONTRIBUTING.md#files-to-change 

I will try as per above steps.

**Reply by nlohmann** [reply]
Do you need further assistance with this issue?
